lib/git-subrepo

```diff
@@ -12,21 +12,8 @@ set -e
 export FILTER_BRANCH_SQUELCH_WARNING=1
 
 # Import Bash+ helper functions:
-SOURCE=${BASH_SOURCE[0]}
-while [[ -h $SOURCE ]]; do
-  DIR=$( cd -P "$( dirname "$SOURCE" )" && pwd )
-  SOURCE=$(readlink "$SOURCE")
-  [[ $SOURCE != /* ]] && SOURCE=$DIR/$SOURCE
-done
-SOURCE_DIR=$(dirname "$SOURCE")
-
-if [[ -z $GIT_SUBREPO_ROOT ]]; then
```

Comment:
GIT_SUBREPO_ROOT is advertised as a valid install method for Windows. See https://github.com/ingydotnet/git-subrepo/issues/271#issuecomment-373722235. Need to investigate if this will work with a Windows system.

Reply:
I imagine, compatibility with Windows is a challenge. I can not say more at this point.